docs/build/unreleased/223.rst

```diff
@@ -0,0 +1,7 @@
+.. change::
+       :tags: usecase, memcached
+       :tickets: 223
+       Added :paramref:`.MemcacheArgs.dead_retry` and
```

Comment:
**mike bayer** (zzzeek) wrote:

add blank line below the ":tickets:" directive before the first paragraph

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/dogpile.cache/+/3930